editor/src/components/inspector/sections/style-section/text-subsection/text-shadow-subsection.tsx

```diff
@@ -175,7 +175,7 @@ const TextShadowItem = React.memo<TextShadowItemProps>((props) => {
     <PropertyRow
       key={props.index}
       style={{
-        gridTemplateColumns: '12px 28px repeat(3, 1fr) 12px',
+        gridTemplateColumns: '12px 28px repeat(3, 1fr) 22px',
```

Comment:
that last 22px is for the button, I assume? It might be nicer to turn this into a template string literal and reference the const for that (we _have_ changed that button size before, and it'll make it clearer what the number means)